sdk/identity/identity/src/credentials/managedIdentityCredential/appServiceMsi2019.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AccessToken, GetTokenOptions, RequestPrepareOptions } from "@azure/core-http";
+import { IdentityClient } from "../../client/identityClient";
+import { credentialLogger } from "../../util/logging";
+import { MSI } from "./models";
+import { msiGenericGetToken } from "./utils";
+
+const logger = credentialLogger("ManagedIdentityCredential - AppServiceMSI 2019");
+
+function expiresInParser(requestBody: any): number {
+  // Parses a string representation of the seconds since epoch into a number value
+  return Number(requestBody.expires_on);
+}
+
+function prepareRequestOptions(resource: string, clientId?: string): RequestPrepareOptions {
+  const queryParameters: any = {
+    resource,
+    "api-version": "2019-08-01"
```

Comment:
Based on the fix for https://github.com/Azure/azure-sdk-for-js/issues/12058, I wonder if we should leave out the 2019 MSI version for now. I made a PR to remove here: https://github.com/Azure/azure-sdk-for-js/pull/12164

cc @schaabs

Reply:
Ok! I'll remove it here too.